### PR TITLE
Require aiopg to be less than 1.3.0

### PR DESCRIPTION
**Pull request by RyanSiu1995**
# Description

Aiopg latest version has breaking changes in its internal object.
It breaks the build in default branch.
I have fixed the version here to make the build pass first.
It has a break changes in its API so we may need a further discussion on how to handle this case.

Fixes # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [X] No.

# Checklist:

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py [hunk 1]
```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("aiopg >= 0.13.0",)
+_instruments = ("aiopg <= 1.2.1",)
```
Comment (owais) [inline instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py:16]: shouldn't this be `>=0.13.0, <1.3.0` consider 1.3 introduced the breaking change and for some reason we did not want to support versions older than 0.13?

Comment (RyanSiu1995) [reply instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py]: I wasn't aware of the support of multiple conditions in python.
I have updated accordingly.

Comment (owais) [reply instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py]: Thanks. Can you please open an issue that tracks adding support for 1.3.0 and newer versions? Please also include details about the issue this instrumentation has today.

Comment (RyanSiu1995) [reply instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py]: Created

**Review by codeboten — APPROVED** [review]

**Review by owais — APPROVED** [review]

**Pull request merged**